### PR TITLE
Add librsb to package index

### PR DESCRIPTION
**Pull request by ivan-pi**
From the [website](http://librsb.sourceforge.net):

> librsb is a library for sparse matrix computations featuring the Recursive Sparse Blocks (RSB) matrix format. This format allows cache efficient and multi-threaded (that is, shared memory parallel) operations on large sparse matrices. The most common operations necessary to iterative solvers are available, e.g.: matrix-vector multiplication, triangular solution, rows/columns scaling, diagonal extraction / setting, blocks extraction, norm computation, formats conversion. The RSB format is especially well suited for symmetric and transposed multiplication variants. Most numerical kernels code is auto generated, and the supported numerical types can be chosen by the user at build time. librsb can also be built serially (without OpenMP parallelism), if required.
>
> librsb also implements the Sparse BLAS standard, as specified in the [[BLAS Technical Forum](http://www.netlib.org/blas/blast-forum/)] documents.

The package fulfills the required criteria:

* Relevance: complete implementation of the Sparse BLAS interface in Fortran
* Maturity: extensive testing has been done over several years
* Availability: http://librsb.sourceforge.net
* Open source: LGPL-3.0 or later
* Uniqueness: rare package for sparse matrix computations with a special storage format and one of less than a handful of Sparse BLAS implementation

cc @michelemartone, would you like to add any tags? I don't think any other [categories](https://github.com/fortran-lang/fortran-lang.org/blob/master/PACKAGES.md#github-hosted-packages) apply.

### CODE REVIEW

**Comment by michelemartone** [comment]
It seems all correct to me.
Maybe as description you could better append " implementing the Sparse BLAS" so to have "A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format implementing the Sparse BLAS" which is a bit more informative.
Thanks @ivan-pi !

File: _data/package_index.yml [hunk 1]
```diff
@@ -254,6 +254,14 @@
   categories: interfaces
   tags: cpp gpu opencl
 
+- name: librsb
+  url: http://librsb.sourceforge.net/
+  description: A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format implementing the Sparse BLAS
```
Comment (ivan-pi) [inline _data/package_index.yml:259]:
```suggestion
  description: A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format implementing the Sparse BLAS standard
```

**Comment by ivan-pi** [comment]
#build_preview

**Review by awvwgk — APPROVED** [review]

**Review by LKedward — APPROVED** [review]

**Comment by LKedward** [comment]
#delete_preview

**Pull request merged**